packages/web/src/scss/components/FileUploader/_FileUploaderInput.scss

```diff
@@ -85,10 +86,54 @@ $_field-name: 'FileUploaderInput';
     display: initial;
 }
 
+:is(.FileUploaderInput--disabled, .FileUploaderInput.is-disabled)
+    :is(.FileUploaderInput__message, [data-element='validator_message']) {
```

Comment:
```suggestion
    :is(.FileUploaderInput__validationText, [data-element='validator_message']) {
```

➕ Plus, to take effect, the disabled styles need to be placed _after_ `form-fields-tools.input-field-validation-states()`.

<img width="330" alt="obrazek" src="https://github.com/lmc-eu/spirit-design-system/assets/5614085/4e1293b5-d66f-4d2b-8e4b-8a519a6ab82f">